displaylink-debian.sh

```diff
@@ -456,7 +456,14 @@ modesetting(){
 test ! -d /etc/X11/xorg.conf.d && mkdir -p /etc/X11/xorg.conf.d
 drv=$(lspci -nnk | grep -i vga -A3 | grep 'in use'|cut -d":" -f2|sed 's/ //g')
 drv_nvidia=$(lspci | grep -i '3d controller' | sed 's/^.*: //' | awk '{print $1}')
-cardsub=$(lspci -nnk | grep -i vga -A3|grep Subsystem|cut -d" " -f5)
+
+# use different content in $cardsub to repair Intel Subsystem handling with kernel >= 6.6  (issue: 920)
+if [ "$(ver2int $kernel_check)" -ge "$(ver2int '6.6')" ];
+then
+    cardsub=$(lspci -nnk | grep -i vga -A3)
```

Comment:
```suggestion
    cardsub=$(lspci -nnk | grep -i vga -A3|grep Subsystem|cut -d" " -f5|cut -d "[" -f2)
```
What do you think about just cutting the trailing bracket instead?
Then we don't have to modify the rest of the file.